plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/order-by-control.tsx

```diff
@@ -63,6 +63,10 @@ const orderOptions = [
 		value: 'menu_order/asc',
 		label: __( 'Manual (menu order)', 'woocommerce' ),
 	},
+	{
+		value: 'random/asc',
```

Comment:
Is there a reason to add the `asc` after the `random`?

Reply:
I don’t think `asc` or `desc` is needed for random order. It’s a minor detail, so feel free to keep it or remove it -it's up to you! 🙂

Reply:
It was needed because [here](https://github.com/woocommerce/woocommerce/blob/6fb3bab6a27668fbd71c202788c5f2ff36199629/plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/order-by-control.tsx#L100-L112) we assume that format will be like `orderBy/order` i.e. `date/asc` or `date/desc`. It will require some refactoring to remove extra "asc" from random sorting option. I was trying to avoid refactoring so as not to break anything. But let me check; it's probably easy to refactor it.